judge/models/problem.py

```diff
@@ -121,8 +121,8 @@ class Problem(models.Model):
                                    validators=[MinValueValidator(settings.DMOJ_PROBLEM_MIN_TIME_LIMIT),
                                                MaxValueValidator(settings.DMOJ_PROBLEM_MAX_TIME_LIMIT)])
     memory_limit = models.PositiveIntegerField(verbose_name=_('memory limit'),
-                                               help_text=_('The memory limit for this problem, in kilobytes '
-                                                           '(e.g. 64mb = 65536 kilobytes).'),
+                                               help_text=_('The memory limit for this problem, in kibibytes '
+                                                           '(e.g. 64mib = 65536 kibibytes).'),
```

Comment:
Might as well take this time to use correct type setting to avoid confusion between MiB and Mib and other craziness.
```suggestion
                                                           '(e.g. 64 MiB = 65536 kibibytes).'),
```